src/site/content/en/learn/css/box-model/self-assessment-1.assess.yml

```diff
@@ -0,0 +1,11 @@
+setLeader: Test your knowledge of the box model
```

Comment:
Oh I'm excited y'all are trying these self assessment widgets out. They were built by one of our contractors who used to work for the national assessment board. Aside from a few places on the site they haven't gotten much use. I'm curious to hear any feedback you have on them.

Reply:
I noticed the widget didn't show up on the page. I think you'd need to add the `{% Assessment %}` shortcode, although maybe you tried and it didn't work?

Reply:
Yeh it didn't work at all and I just gave up lol. I think maybe the client web components didn't load?

Reply:
That is entirely possible, I'll take a look